### ISSUE TITLE
Put query end of statement period within OPTIONALs in all queries

### ISSUE THREAD

**Issue by andrewtavis** [scribe-org/Scribe-Data#391]
### Terms

- [X] I have searched [open and closed issues](https://github.com/scribe-org/Scribe-Data/issues?q=is%3Aissue)
- [X] I agree to follow Scribe-Data's [Code of Conduct](https://github.com/scribe-org/Scribe-Data/blob/main/.github/CODE_OF_CONDUCT.md)

### Issue

As of now there are many queries in Scribe-Data where the `OPTIONAL` selections has a period at the end of it. It would be ideal if the period was at the end of the statement within it instead in all cases. So rather than:

```sparql
  OPTIONAL {
    ?lexeme ontolex:lexicalForm ?absSingularForm .
    ?absSingularForm ontolex:representation ?absSingular ;
      wikibase:grammaticalFeature wd:Q332734, wd:Q110786 ;
  } .
```

Rather:

```sparql
  OPTIONAL {
    ?lexeme ontolex:lexicalForm ?absSingularForm .
    ?absSingularForm ontolex:representation ?absSingular ;
      wikibase:grammaticalFeature wd:Q332734, wd:Q110786 .
  }
```

This issue is relatively laborious, but it would be really great to get someone to pick it up and fix all of the queries 😊

**Comment by kanishk128** [comment]
Hey, I could help out with this, could it be assigned to me?

**Comment by andrewtavis** [comment]
Sounds great, @kanishk128! Thanks for your offer to help :)

**Comment by kanishk128** [comment]
Hey, completed it @andrewtavis! Was pretty cumbersome, skipped my dinner unknowingly. Check and revert for feedback if any

**Comment by andrewtavis** [comment]
Closed by #393 :) Ya I can imagine, @kanishk128... Appreciate you putting in all of the effort! The point of this is actually making sure that the queries are uniform so that we can parse them for correctness automatically later. Thanks a lot! 😊
